packages/appstore/src/components/transaction-list/transactions-for-a-day/index.ts

```diff
@@ -0,0 +1,3 @@
+import TransactionsForADay from './transactions-for-a-day';
```

Comment:
```suggestion
import TransactionsForADay from './transactions-for-a-day';
```
isit possible to change the filename to `daily-transactions` or anything else with less `-`?

Reply:
Sure, ofc 😅

Reply:
Done